latch_metadata/__init__.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+from latch.types.directory import LatchDir
+from latch.types.metadata import (
+    LatchAuthor,
+    NextflowMetadata,
+    NextflowRuntimeResources,
+)
+
+from .parameters import flow, generated_parameters
+
+NextflowMetadata(
+    display_name="nf-core/ampliseq",
```

Comment:
I feel like we should change display names to NF-Core Ampliseq

basically just remove the `/` kinda feels ugly but we can stick to whatever convention nf-tower uses b/c that is what people will be familiar with